### DISCUSSION TITLE
How do I convince my colleagues to use quarto?

### DISCUSSION THREAD

**Discussion by rhydar** [quarto-dev/quarto-cli#7826] [Show and Tell]
### Description

Hi. I've been using some form of reproducible research principles for around 10 years. Started with Sweave, then moved to knitr. With Quarto and being able to seamlessly integrate python and R into live/traceable documents, I feel that Quarto is now a very a viable option within my industry (Maritime Engineering Consultancy) for document generation within projects. I've demonstrated a couple of qmd files in the office, but, I'm not quite getting the reactions or the traction that I expected. 
Long story short. How do I go about pitching the benefits of quarto (and reproducible/traceable research) within my industry or outside of academia/statistics?

**Comment by mcanouil** [comment]
If your main outputs are Word and PDF documents and that the argument, about which Quarto allows to include computation directly in the rendered document without copying/pasting anything, does not convince, I don't see what could.

If your main output is HTML based documents, then a good argument that usually work very well, is interactivity either directly for the computing languages (R/Python) but also via Quarto with Observable.

You could look at the recordings of Quarto introduction in R User group and conferences to get idea of highlight points you could use to target specifically your colleagues.

**Comment by harrelfe** [comment]
In my experience `Quarto` sells itself, but wow your colleagues with beautiful output including marginal notes/graphs/tables.  Examples: https://hbiostat.org/bbr , https://hbiostat.org/rflow .

**Comment by jimjam-slam** [comment]
I think it really depends on where your colleagues are coming from. If they use Jupyter notebooks, you might want to focus on the language agnosticism that Quarto enables, or on We're on the reproducibility concerns that Quarto and Rmarkdown sidestep. If they're already using Rmarkdown, then showing off layout features or OJS might be the go.

If they're using Word and Excel, then doing a post-mortem on a graphic that had to be repeatedly built in Excel to accommodate changes in upstream data, or on a report where content was lost between revisions, might help your colleagues appreciate the value (in time and money) of version control and reproducible graphics.

Good luck!

**Comment by baptiste** [comment]
> I'm not quite getting the reactions or the traction that I expected

Maybe the best strategy would be to ask a few colleagues directly (selected for their frank but open-minded feedback), why they're not convinced/enthusiastic. It might be something entirely unrelated to Quarto's capabilities – maybe they do find it impressive but to them it looks very complicated (e.g. they're not familiar with plain text editors, or the command line, etc.)

Having those discussions might suggest practical steps to get more people excited, such as teaming up on a small project with someone who's not yet a quarto user.

**Comment by rhydar** [comment]
Thanks all for the responses. A lot of the issues with uptake I believe is the perception that it's **difficult**. I use the VS Code Quarto extension and mostly had to setup dev-containers for quarto to bundle the dependencies, this makes a fairly easy one stop shop, but even  then, repeatability is sacrificed for ease of starting (i.e. a docx) and cut and paste figures. One consistent statement that I hear against using text base solutions is the ability for docx docs to be edited/monitored by multiple engineers concurrently and changes tracked automatically. I've had a cursory look at some live sharing extensions in VS Code, and use version control, but the latency it problematic. Once again, I appreciate the inputs.

**Reply by allefeld** [reply]
Track Changes is really the one big thing which Word & Co have over Markdown-based workflows.

- There is [CriticMarkup](https://fletcher.github.io/MultiMarkdown-6/syntax/critic.html), but if there are extensive changes that just creates unreadable Markdown.

- I don't think Markdown markup for changes is the right way to go, that should be metadata managed by the editor. But then of course the advantage of Markdown that any text editor can be used is gone. This is basically the path [Overleaf](https://www.overleaf.com/) went regarding LaTeX.

- In principle git-based tracking of changes is an alternative, but I have to admit that even my quite nerdy self isn't very comfortable with git merges, let alone collaborators.

- In an academic context where there is a clearly designated primary author, an alternative is to create Word from Markdown, let collaborators edit & comment on the docx, and then to enter the changes manually in the Markdown source – this way ease of use is at least preserved for others, though not for me.
